includes/utils.inc

```diff
@@ -1057,6 +1057,23 @@ function wf_crm_get_fields($var = 'fields') {
         'name' => t('Contribution Source'),
         'type' => 'textfield',
       );
+      // KG - start
+      // LineItems
+      $sets['lineItems'] = array('entity_type' => 'line_item', 'label' => t('Line Items'));
```

Comment:
I feel like the entity type should be 'contribution' but not 100% sure.

Reply:
Also feel like this line belongs at the top with the other line items.

Reply:
And should be conditional with CiviContribute being enabled :)

Reply:
I'll have another look at the entity type [I did start off with 'contribution'. Not sure what made me change it]; and yes I can move that Set + make it conditional.

Reply:
I was able to move the Set to the top though - working on making it conditional now.

Reply:
Almost done. Easier if it was $ent = 'contribution' - going to have one more pass at looking into that.